mypyc/test-data/run-classes.test

```diff
@@ -2741,7 +2741,9 @@ def test_buffer_roundtrip() -> None:
     write_tag(b, TAG_B)
     write_int(b, 2)
     write_int(b, 2 ** 85)
+    write_int(b, 255)
     write_int(b, -1)
+    write_int(b, -255)
```

Comment:
Test also the edge cases (-11, -10, -9, 116, 117, 118). Test a few more different lengths of integers (e.g. 15 bits, 23 bits, 30 bits) with arbitrary lower bits.

Reply:
OK, I think it would also make sense to test something like `len(data.getvalue()) == 1` etc.